Confus/Confus/Game_BACKUP_1736.cpp

```diff
@@ -0,0 +1,133 @@
+    }
+
+    void Game::processTriangleSelectors()
+    {
+        auto sceneManager = m_Device->getSceneManager();
+        auto metatriangleSelector = sceneManager->createMetaTriangleSelector();
+        
+        irr::core::array<irr::scene::ISceneNode*> nodes;
+        sceneManager->getSceneNodesFromType(irr::scene::ESNT_ANY, nodes, m_LevelRootNode);
+        for(irr::u32 i = 0; i < nodes.size(); ++i)
+        {
+            irr::scene::ISceneNode* node = nodes[i];
+            irr::scene::ITriangleSelector* selector = nullptr;
+
+            switch(node->getType())
+            {
+            case irr::scene::ESNT_CUBE:
+            case irr::scene::ESNT_ANIMATED_MESH:
+                selector = m_Device->getSceneManager()->createTriangleSelectorFromBoundingBox(node);
+                break;
+            case irr::scene::ESNT_MESH:
+            case irr::scene::ESNT_SPHERE:
+                selector = sceneManager->createTriangleSelector(((irr::scene::IMeshSceneNode*)node)->getMesh(), node);
+                break;
+            case irr::scene::ESNT_TERRAIN:
+                selector = sceneManager->createTerrainTriangleSelector((irr::scene::ITerrainSceneNode*)node);
+                break;
+            case irr::scene::ESNT_OCTREE:
+                selector = sceneManager->createOctreeTriangleSelector(((irr::scene::IMeshSceneNode*)node)->getMesh(), node);
+                break;
+            default:
+                break;
+            }
+
+            if(selector)
+            {
+                metatriangleSelector->addTriangleSelector(selector);
+                selector->drop();
+            }
+        }
+        m_LevelRootNode->setTriangleSelector(metatriangleSelector);
+    }
+
+    void Game::handleInput()
+    {
+        m_PlayerNode.handleInput(m_EventManager);
+    }
+
+    void Game::update()
+    {
+        m_PreviousTicks = m_CurrentTicks;
+        m_CurrentTicks = m_Device->getTimer()->getTime();
+        m_DeltaTime = (m_CurrentTicks - m_PreviousTicks) / 1000.0;
+
+        m_PlayerNode.update();
+    }
+
+    void Game::processFixedUpdates()
+    {
+        m_FixedUpdateTimer += m_DeltaTime;
+        m_FixedUpdateTimer = irr::core::min_(m_FixedUpdateTimer, MaxFixedUpdateInterval);
+        while(m_FixedUpdateTimer >= FixedUpdateInterval)
+        {
+            m_FixedUpdateTimer -= FixedUpdateInterval;
+            fixedUpdate();
+        }
+    }
+
+    void Game::fixedUpdate()
+    {
+<<<<<<< HEAD
+		static float timer = 0.0f;
+		timer += static_cast<float>(m_DeltaTime);
+		if (timer >= 9.0f)
+		{
+			timer = 0.0f;
+		}
+		m_MazeGenerator.fixedUpdate();
+=======
+        m_MazeGenerator.fixedUpdate();
+>>>>>>> develop
+    }
+
+    void Game::render()
+    {
+        m_Device->getVideoDriver()->beginScene(true, true, irr::video::SColor(255, 100, 101, 140));
+        m_Device->getSceneManager()->drawAll();
+        m_Device->getGUIEnvironment()->drawAll();
+        m_Device->getVideoDriver()->endScene();
+    }
+}
```

Comment:
Could you remove this file along with the other backup files? You can lookup online how you can prevent git from creating these backup files